services/kafka/service.go

```diff
@@ -210,7 +224,8 @@ func configChanged(old, new Config) bool {
 	return old.UseSSL != new.UseSSL ||
 		old.SSLCA != new.SSLCA ||
 		old.SSLCert != new.SSLCert ||
-		old.SSLKey != new.SSLKey
```

Comment:
Have we tested that the config change for kafka works properly (that the new producer is correctly restarted?)

Edit: I see the automated test below but this seems like a big enough change to manually try it out as well.